config/agent-spawner/config.yaml

```diff
@@ -9,12 +9,14 @@
 #    # agent immediately marked for removal.  `cmd_terminate` exit status is
 #    # just ignored (we need to remove the agent no matter what).
 #
-#    # Prepare the agent.  Variable $RESALLOC_RESOURCE_DATA (base64 encoded)
-#    # is provided in the script environment.  Other variables like
-#    # RESOURCE_NAME, RESOURCE_POOL_ID, etc. are provided as well.
+#    # Prepare the agent.  Variable $AGENT_SPAWNER_RESOURCE_DATA (base64
```

Comment:
The only thing I may add is that base64 decoded data may contain new lines. It could be tricky to debug, if you are not familiar with functioning of resalloc. It kept me confused for a while.

Reply:
Indeed, the current code doesn't modify the stdout at all, and thus it contains the trailing newline.  We could implement some RFE for stdout post-processing if you think it is worth it.